awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/Toolbar.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
For some reason, I'm always seeing a node count of 0 on the toolbar:
![Screenshot from 2019-11-22 12-11-06](https://user-images.githubusercontent.com/9753817/69445940-605cd580-0d21-11ea-9974-904274546970.png)

Reply:
Yea thats expected for now - nothing in the toolbar is functional